pkgs/applications/audio/sublime-music/default.nix

```diff
@@ -28,9 +28,12 @@ python3Packages.buildPythonApplication rec {
   nativeBuildInputs = [
```

Comment:
```suggestion
  nativeBuildInputs = with python3Packages; [
```

Reply:
I don't think this is very relevant and I am not comfortable doing style changes (even if I largely agree with them) to packages I hold no stake in. As long as we don't have any published standards that should in my principled opinion be left to the "owners" of the package.